test_app.py

```diff
@@ -1,4 +1,7 @@
 import unittest
+import sys
+from io import StringIO
+from tictactoe import *
```

Comment:
The [general opinion](
https://docs.python.org/3/tutorial/modules.html#importing-from-a-package) on this style of import is that it can be a little dangerous to the global symbol table and can cause unintended namespace clashes when your application gets bigger.

Reply:
Noted, only importing the required class from now on. Since the TicTacToe class has been refactored to reduce its workload, there are now 3 test files, importing only their respective classes.